src/test/java/bio/terra/service/filedata/google/gcs/GcsPdaoUnitTest.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.service.filedata.google.gcs;
+
+import bio.terra.common.category.Unit;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Category(Unit.class)
+public class GcsPdaoUnitTest {
```

Comment:
It seems like it would be better to have all tests for `GcsPdao` in a single class. Can the `@Category` annotation be used on the test methods?

Reply:
I think we have to keep them separate since fundamentally the class lifecycle is different (integration tests often have class initialization logic around setting up). I prefer keeping these separae